apps/io-services-cms-webapp/OnRequestPublication/function.json

```diff
@@ -0,0 +1,13 @@
+{
+  "bindings": [
+    {
+      "queueName": "request-publication",
```

Comment:
```suggestion
      "queueName": "%REQUEST_PUBLICATION_QUEUE%",
```

Reply:
@giuseppedipinto can you please do the same change to `OnRequestReview/function.json` (i forgot about that)? Thanks

PS: if you want to add more control, you can check for the existence of these variables in config.ts 😉

Reply:
c00ca73